cl/recap_email/app/app.py

```diff
@@ -180,6 +179,7 @@ def handler(event, context):  # pylint: disable=unused-argument
     gray_or_pass_tests = (
         get_spf_verdict,
         get_dmarc_verdict,
+        get_dkim_verdict,
```

Comment:
Hm, instead of removing this, would it be better to allow this one to be gray while not allowing the other tests to be gray?

Reply:
I only moved get_dkim_verdict from the tests that must be "PASS" to the existing tests that could be "PASS" or "GRAY".

I assume that get_spf_verdict and get_dmarc_verdict also could be GRAY in some situations.

Reply:
Oh, I mis-read the diff. Nm. Merging, thanks.